src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/Example1.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="JavadocMethod"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+// xdoc section -- start
+public class Example1 {
+
+  /** */
+  Example1(int x) {}
+  // violation above, 'Expected @param tag for 'x''
+  /** */
+  public int testMethod1(int p1) {
+    // violation above, '@return tag should be present'
+    return p1; // violation 2 lines above 'Expected @param tag for 'p1''
+  }
```

Comment:
please make body of all methods to be single line, their content does not matther
```
  /** */
  public int m1(int p1) { return p1; }
  // 2 violations above:
  //    '@return tag should be present'
  //    'Expected @param tag for 'p1''
```

please use "mX" for name of of method to keep them short vertically.
plesae use `// X violations above|below:`

Reply:
Done